LEGO1/lego/legoomni/include/registrationbook.h

```diff
@@ -33,6 +33,23 @@ class RegistrationBook : public LegoWorld {
 
 	// SYNTHETIC: LEGO1 0x10076f30
 	// RegistrationBook::`scalar deleting destructor'
+protected:
```

Comment:
for now we can assume these are private since nothing else needs access at the moment